src/app/frontend/deploy/createnamespace.html

```diff
@@ -16,12 +16,12 @@
 
 <md-dialog aria-label="Create a new namespace" layout="column" layout-padding>
   <md-content layout-padding>
```

Comment:
That explains a lot! I've changed `md-content` into `md-dialog-content` in all dialogs. `layout-padding` is used now only fot `md-dialog-content`, no longer for `md-dialog`. I've reverted my CSS changes. Also used `<div>` instead of `<p>` to avoid additional paddings.